packages/x-date-pickers/src/PickersLayout/PickersLayout.tsx

```diff
@@ -81,7 +81,7 @@ export const PickersLayoutContentWrapper = styled('div', {
   slot: 'ContentWrapper',
   overridesResolver: (props, styles) => styles.contentWrapper,
 })<{ ownerState: PickerLayoutOwnerState }>({
-  gridColumn: 2,
+  gridColumn: '2 / 4',
```

Comment:
This seemed like a mistake. 🤔

Reply:
@alexfauquette if you remember :laughing: